microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay.md

```diff
@@ -15,39 +15,40 @@ While it is possible for PWAs on mobile to define how they want to be displayed
 
 By default, the app area starts right below the reserved title bar area.
```

Comment:
change to "desktop PWAs can't use this APPROACH to create an immersive, native-like, experience"